maps-block-apple.php

```diff
@@ -25,20 +25,47 @@
 define( 'MAPS_BLOCK_APPLE_BASENAME', plugin_basename( __FILE__ ) );
 
 /**
- * Require WP version >=5.8
- * PHP 7.4 errors should be caught in the sandbox during activation.
+ * Get the minimum version of PHP required by this plugin.
+ *
+ * @return string Minimum version required.
  */
-register_activation_hook(
```

Comment:
Do we want to get rid of this check? This seems different from the PHP version check we are adding here

Reply:
We are not loading the plugin functionality if minimum php version requirement does not match, and we show the admin notice for the same. 
I thought this check on activation looked redundant, Since we will still see the admin notice / error on plugin activation if php version didn't match the requirement, and plugin won't load.

Reply:
So the check we're removing isn't looking at the PHP version, it's looking at the WordPress version (it shows an error if using WordPress <5.8. I'm not sure that check is needed but also want to ensure we're not getting rid of it if it is needed. Thinking we may want to add a WordPress `version_compare` within our new conditional so we keep that check. Something like:

```php
if ( ! site_meets_php_requirements() || ! version_compare( $GLOBALS['wp_version'], '5.8', '>=' ) ) {
```

Reply:
Or another approach is how we do this in Distributor: https://github.com/10up/distributor/blob/develop/distributor.php#L93